lib/iris/analysis/maths.py

```diff
@@ -35,6 +35,8 @@
 import iris.exceptions
 import iris.util
 
+import biggus
```

Comment:
Having imported Biggus it doesn't look like you're then using it... Is it used for accessing `cube._my_data`?